terraform/environments/oasys/locals_test.tf

```diff
@@ -93,6 +93,25 @@ locals {
       }
     }
 
+    baseline_iam_policies = {
+      Ec2T2BipPolicy = {
+        description = "Permissions required for T2 Bip EC2s"
+        statements = [
+          {
+            effect = "Allow"
+            actions = [
+              "secretsmanager:GetSecretValue",
+            ]
+            resources = [
+              "arn:aws:secretsmanager:*:*:secret:/oracle/bip/t2/*",
+              "arn:aws:secretsmanager:*:*:secret:/oracle/database/*T2/bip-*",
+              "arn:aws:secretsmanager:*:*:secret:/oracle/database/T2*/bip-*",
```

Comment:
i was wondering for simplicity sake having (in baseline_presets)
`"arn:aws:secretsmanager:*:${var.environment.account_id}:secret:/oracle/bip/*",`
and 
`"arn:aws:secretsmanager:*:${var.environment.account_id}:secret:/oracle/database/*",`
in 2 policies. I have a PR for it

that would give options to get access to either or both bip and database passwords in the account it is in, however the downside is it would give all test environments access to all db or bip secrets in test. But for preprod and prod, i don't think it'd be a problem. I like that it'd be simple. Thoughts?